esp8266/qstrdefsport.h

```diff
@@ -42,11 +42,23 @@ Q(sync)
 Q(hard_reset)
 
 Q(esp)
+Q(esp.wifi)
```

Comment:
If it works, it's kinda bug, and likely against Python semantics. If it's useful, worth thinking.

P.S. Actually, the only way it might work is if parser would parse it as one qstr. Does it? Again, that would be a bug in parser ;-).